client/src/app/_services/member.service.ts

```diff
@@ -25,6 +27,10 @@ export class MemberService {
   }
 
   delete(id: Number, code: Number): Observable<Member> {
-    return this.http.post<Member>(`/api/members/${id}`, { code });
+    return this.http.post<Member>(`/api/members/${id}/delete`, { code });
+  }
+
+  register(id: Number, code: Number): Observable<Registration> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this 'Number' wrapper object with primitive type 'number'. ([typescript:S1533](https://sonarcloud.io/organizations/k-fet/rules#rule_key=typescript%3AS1533))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=k-app&pullRequest=282&issues=AWUvVpLZWWqqPQVjCkiv&open=AWUvVpLZWWqqPQVjCkiv)

[ISSUE_KEY]: AWUvVpLZWWqqPQVjCkiv